### DISCUSSION TITLE
Better Texture Quality

### DISCUSSION THREAD

**Discussion by Montrazul** [SFX-WoW/Masque#184] [Questions] [answered]
Hi,

this is not an issue. I just have a question and didnt find something else to ask it.
I created a very simple masque texture here: https://wow.curseforge.com/projects/masque-lybrial

As you can see on the Screenshots the resolution of the texture is terrible. 
Is there a way to get / create a better quality?

**Comment by StormFX** [comment, marked as answer]
Most of my textures are 64 x 64. Make sure you're using decent image-editing software. You won't get good results with MS Paint, etc.

**Comment by Montrazul** [comment]
Did it with Gimp.
